imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -152,13 +159,18 @@ class CaseExplorer extends Component {
                 />)
               }
               name={'case'}
-            /> : (<NoItemMsg item={'case'} />)
+            /> : (<NoItemMsg item={'case'} ButtonOption />)
```

Comment:
This doesn't work because you didn't rename it when you renamed the component's prop to `buttonOption`